docker-compose-base.yml

```diff
@@ -33,3 +33,5 @@ server:
     - FXA_SECRET_KEY=a93644d71fa684da464fd2dba4d687c14cb21e1dd740af83895e70ede82f54cc
     - DATADOG_API_KEY=4f3b967bbf13c7769ac4fa89efda0fae
     - DATADOG_APP_KEY=ddd45a7b3a3cb90ff1baf20ae8cfab04d1937038
+    - ACCOUNT_INVITE_ONLY_MODE=True
+    - MOZILLIANS_API_KEY=ff33fa1255ce0ad59be17271093ab851c6357936
```

Comment:
Sorry, ignorant question here, but are all these KEYS kosher in GitHub? Part of me mildly panics when I see `x_SECRET_KEY` or thinking that somebody could use our `MOZILLIANS_API_KEY` for their apps (although not really an issue if the keys only work for certain pre-approved URLs).